tests/test_cli.py

```diff
@@ -32,88 +33,94 @@ def test_geojson_invalid_second_input(script_runner):
 def test_geojson_bbox(script_runner):
     ret = script_runner.run('geoextent',
         '-b',
-        'tests/testdata//geojson/muenster_ring_zeit.geojson')
+        '-input=', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
     assert ret.success, "process should return success"
     assert "[7.60168075561523, 51.9488147720619, 7.64725685119629, 51.9746240298775]" in ret.stdout, "bbox is printed to console"
 
 def test_geojson_bbox_long_name(script_runner):
     ret = script_runner.run('geoextent',
-        '--bounding-box', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
+        '--bounding-box', '-input=', 'tests/testdata//geojson/muenster_ring_zeit.geojson')
```

Comment:
Is the `-input=` needed now? It should not be, so I suggest to leave some of the tests without it.

Reply:
Yes, it is needed.

Reply:
If I update it to previous impl. the `-formats` won't work and going to raise an error.